shell/platform/embedder/embedder.cc

```diff
@@ -46,18 +48,22 @@ const int32_t kFlutterSemanticsNodeIdBatchEnd = -1;
 const int32_t kFlutterSemanticsCustomActionIdBatchEnd = -1;
 
 static FlutterEngineResult LogEmbedderError(FlutterEngineResult code,
-                                            const char* name,
+                                            const char* reason,
+                                            const char* code_name,
                                             const char* function,
                                             const char* file,
                                             int line) {
-  FML_LOG(ERROR) << "Returning error '" << name << "' (" << code
-                 << ") from Flutter Embedder API call to '" << function
-                 << "'. Origin: " << file << ":" << line;
+  const auto file_base = (::strrchr(file, '/') ? strrchr(file, '/') + 1 : file);
```

Comment:
Are the paths handed to us in file normalised? i.e. will this be an issue on Windows?

Reply:
No. Yes. See https://godbolt.org/z/F6SO6q. Fixed.